orm/src/main/java/org/hibernate/search/jpa/FullTextEntityManager.java

```diff
@@ -42,10 +42,13 @@
 
 	/**
 	 * Force the (re)indexing of a given <b>managed</b> object.
-	 * Indexing is batched per transaction: if a transaction is active, the operation
+	 * Indexation is batched per transaction: if a transaction is active, the operation
```

Comment:
I think I just copy/pasted the comments from FullTextSession, which I guess was more recent. I failed to see this difference.